app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -50,6 +50,8 @@ function YourTurnContent(props: {
   activeTurn: CurrentGameSubscription["games"][0]["turns"][0]
   activeTurnPlayState: ActiveTurnPlayState
   secondsLeft: number
+  currentRoundId: number
+  nextRoundId: number
```

Comment:
don't think these should be typed as numbers?

probably `Round["id"]`

Reply:
Ah the issue is uuid is typed as "any" :'(  ... still let's update to above suggestion imo

Reply:
yep, I looked for a `uuid` type but didn't see any precedent for it in this PR (though i came across [this library](https://www.npmjs.com/package/@types/uuid), not sure if you're familiar with it).

forgot about the `Round["id"]` trick, will update